### PR TITLE
+ kafka.0.2

### PR DESCRIPTION
**Pull request by ygrek**

### CODE REVIEW

File: packages/kafka/kafka.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+]
+build-test: [make "test"]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "kafka"]
+depopts: ["lwt"]
+depends: [
+  "ocamlfind" {build}
+]
+depexts: [
+  [["debian"] ["librdkafka-dev"]]
```
Comment (whitequark) [inline packages/kafka/kafka.0.2/opam]: [["ubuntu"] ["librdkafka-dev"]]

if you want the Travis build to succeed.

Comment (ygrek) [inline packages/kafka/kafka.0.2/opam]: fixed

**Comment by whitequark** [comment]
```
# ocaml_kafka.c:874:6: warning: implicit declaration of function ‘rd_kafka_metadata_destroy’ [-Wimplicit-function-declaration]
# ocaml_kafka.c: In function ‘ocaml_kafka_get_topics_metadata’:
# ocaml_kafka.c:899:18: error: dereferencing pointer to incomplete type
# ocaml_kafka.c:899:18: error: request for member ‘topic_cnt’ in something not a structure or union
# ocaml_kafka.c:899:8: warning: assignment makes integer from pointer without a cast [enabled by default]
# ocaml_kafka.c:900:6: error: unknown type name ‘rd_kafka_metadata_topic_t’
# ocaml_kafka.c:900:49: error: dereferencing pointer to incomplete type
# ocaml_kafka.c:900:49: error: request for member ‘topics’ in something not a structure or union
# ocaml_kafka.c:900:41: warning: initialization discards ‘const’ qualifier from pointer target type [enabled by default]
```

**Comment by ygrek** [comment]
Do you know what version of librdkafka-dev is there? It builds fine on debian with 0.8.5

**Comment by ygrek** [comment]
I can see it is 0.8.3. But `rd_kafka_metadata_topic_t` first appeared in 0.8.4

**Comment by ygrek** [comment]
ubuntu is so outdated, as usual :)

**Pull request merged**